Messenger/AppDelegate.mm

```diff
@@ -755,6 +756,18 @@ - (WebView *)webView:(WebView *)sender createWebViewWithRequest:(NSURLRequest *)
 }
 
 
+- (NSArray *)webView:(WebView *)sender
+contextMenuItemsForElement:(NSDictionary *)element
+    defaultMenuItems:(NSArray *)defaultMenuItems {
+  #if DEBUG
+    if (([NSEvent modifierFlags] & NSControlKeyMask) == NSControlKeyMask) {
```

Comment:
I used to do that, but then I realized that implicitly casting a bitmask to an ObjC BOOL (which is an unsigned char) can fail if it's happening through a method argument because only the lowest bit, or lower 8 bits will be checked (depending on the compiler, optimization level, etc.).

In this case, it's just a C conditional, so any value other than 0 is fine. I'll change it.